src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -204,6 +204,28 @@ class PythonSetup(Subsystem):
         ),
         advanced=True,
     )
+    no_binary = StrListOption(
+        "--no-binary",
+        help=(
+            "Do not use binary packages (i.e., wheels) for these 3rdparty distributions. Also "
+            "accepts :all: to disable all binary packages. Note that some packages are tricky to "
```

Comment:
I didn't bother documenting :none: since we have our own mechanism for modifying list options, and this would just get more confusing.